codesign/test/file_codesing_visitor_test.dart

```diff
@@ -37,6 +37,7 @@ void main() {
         tempDir: tempDir,
       );
       codesignVisitor.directoriesVisited.clear();
+      codesignVisitor.initialize();
```

Comment:
Is there any reason why the initialization code isn't run in the visitor's constructor? This seems like an easy candidate for a future bug, where someone forgets to call initialize before using the visitor, or it gets called twice.

Reply:
True, I forgot why i had it separately initialized. Moved.